tests/utils.go

```diff
@@ -779,23 +768,14 @@ func AddEphemeralCdrom(vmi *v1.VirtualMachineInstance, name string, bus v1.DiskB
 	return vmi
 }
 
-func NewRandomFedoraVMI() *v1.VirtualMachineInstance {
+func NewRandomFedoraVMI(opts ...libvmi.Option) *v1.VirtualMachineInstance {
```

Comment:
nit: in the long term we should remove all the [man-in-the-middle funcs](https://www.refactoring.com/catalog/removeMiddleMan.html) here entirely.

Reply:
Agree, but IMHO it will be a little out of the scope of this PR. I think it will be great to do it in a followup PR. WDYT? Thanks